db/app.js

```diff
@@ -0,0 +1,15 @@
+const express = require("express");
+const app = express();
+const { handlePsqlErrors, handleCustomErrors, handleServerErrors } = require("./error");
+const { getAPI, getAllTopics, routeNotFound } = require("./app.controller");
+app.use(express.json());
```

Comment:
Do you need express.json yet? Only push code that is relevant to the ticket please!